RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -6105,3 +6105,43 @@ float mp_regen()
 {
 	return 0.5 * (numeric_modifier("MP Regen Min") + numeric_modifier("MP Regen Max"));
 }
+
+boolean auto_canForceNextNoncombat()
+{
+	return auto_pillKeeperAvailable()
+	|| (!get_property("_claraBellUsed").to_boolean() && (item_amount($item[Clara\'s Bell]) > 0))
```

Comment:
do we need an auto_is_valid for Clara's as well? i suppose we technically wouldnt have one unless it was valid but might be nice to have the check as well for completeness